syllabus.md

```diff
@@ -11,20 +11,39 @@ The course begins with an introduction to probability theory, with pencil-and-pa
 
+- **Solution Files** - Solutions to each homework and each live session will be available on [[Github]](https://github.com/mids-w203). You learning is not complete when you solve a problem; it is important that you view the solution to solidify your knowledge and identify ways to improve your arguments. Homework solutions are located in the semester's "central" repository, and will be updated Thursdays at 2pm. Each live session repository will be located in a separate repository, with a name like unit_XX_ls_sol. This allows us to give each student access to the solution immediately after live session. 
+
+- **Tests** - The course includes two tests, located on [[Gradescope]](www.gradescope.com). The tests are made available during week 4 and week 6 of the course, and must be completed by Tuesday 2pm of the following week. There is a 3-hour time limit for each test. It is up to you to decide when to begin each test. Once you start the test on [[Gradescope]](www.gradescope.com), a 3-hour timer will begin to count down. Practice tests will be available to give you a sense of what the tests are like.
+
+- **Labs** - There are two labs that are multi-week, group-projects. Labs are available on [[Github]](https://github.com/mids-w203) and will be submitted via [[Gradescope]](www.gradescope.com). Groups are typically between three and four students. Each lab includes a complete analysis, from data wrangling and statistical procedures, to writing a persuasive report.
+
+- **Slack Channel** - We use a slack channel, datasci-203-2022-spring, for official course announcements and general discussion. This is a public channel within the ucbischool.slack.com workspace.
+
+- **Practice Problems** - The course includes a set of curated [[practice problems]](http://w203.herokuapp.com/). Use your slack credentials to log in. Each problem comes with a comment functionality that you can use to share and discuss solutions. While optional, we highly recommend solving lots of practice problems each week. This is the best way to get really comfortable with probability theory and great preparation for the tests.
+
+- **Participation** - Participation is more than just showing up to class.  It includes being prepared, engaging actively, and treating others with respect to sustain a thriving learning environment.
+
+## Assignment Submission Guidelines
+
+Mathematical exercises may be typeset using Latex, or hand-written and scanned, but you must ensure that the final file is easy to read. Instructors will not grade submissions with unusual file types or formatting that makes access difficult.
```

Comment:
For early assignments (Hw 1-5 and test 1), it is acceptable to do the work with pencil and paper. the submission can be photographed, then combined together as a PDF. the PDF can be submitted to gradescope. For later assignments, we will move past hand-written submissions.

Reply:
maybe we should ask @blulightspecial if the wording fits with grading needs.